Lib/fontmake/instantiator.py

```diff
@@ -321,17 +317,15 @@ def collect_glyph_masters(
     Note: skips empty source glyphs if the default glyph is not empty to almost match
     what ufoProcessor is doing. In e.g. Mutator Sans, the 'S.closed' glyph is left
     empty in one source layer. One could treat this as a source error, but ufoProcessor
-    specifically has code to skip that empty glyph and carry on.
+    specifically has code to skip that empty glyph and carry on. The code might be
+    there to support the SourceDescriptor.mutedGlyphNames attribute that clears the
+    glyph from the master that specifies it, we don't support that attribute though.
```

Comment:
we don't need to be so verbose here, i think you can remove this paragraph.

Reply:
Maybe for archeology purposes? :)

Reply:
comments like these will get "rotten" very quickly